yarn-project/end-to-end/src/e2e_orderbook.test.ts

```diff
@@ -0,0 +1,138 @@
+import { type AccountWallet, type FieldLike, Fr, type Logger, type PXE } from '@aztec/aztec.js';
+import { type OrderCreated, type OrderFulfilled, OrderbookContract } from '@aztec/noir-contracts.js/Orderbook';
+import type { TokenContract } from '@aztec/noir-contracts.js/Token';
+
+import { jest } from '@jest/globals';
+
+import { deployToken, mintTokensToPrivate } from './fixtures/token_utils.js';
+import { setup } from './fixtures/utils.js';
+
+const TIMEOUT = 120_000;
+
+// We test only a happy path here since this is just a demonstration of the orderbook contract and writing here more
+// thorough tests seems unnecessary. Also hopefully we will migrate this to TXE eventually. Didn't write it in TXE
+// now as there is no way to obtain public events and all of TXE will be rewritten soon.
```

Comment:
Can you open an issue? This is a good example test to have, since it involves multiple parties, events, etc., and requires little setup.